pom.xml

```diff
@@ -301,12 +301,9 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.7.0</version>
+                    <version>3.8.0</version>
                     <configuration>
                         <compilerId>javac-with-errorprone</compilerId>
-                        <compilerArgs>
```

Comment:
❓ Are _all_ the junit issues that errorprone finds and reports now resolved?

Reply:
The build runs correctly, but that flag no longer works on the new bundle plugin. I resolved all of the issues that that flag was hiding that caused the build to break.